spikeinterface/sortingcomponents/tests/test_peak_detection.py

```diff
@@ -29,7 +29,7 @@ def test_detect_peaks():
         repo=repo, remote_path=remote_path, local_folder=None)
     recording = MEArecRecordingExtractor(local_path)
     
-    job_kwargs = dict(n_jobs=2, chunk_size=10000, progress_bar=True, verbose=True)
+    job_kwargs = dict(n_jobs=-1, chunk_size=10000, progress_bar=True, verbose=True)
```

Comment:
This should make the test use paralell in the CI and also run in the unlikely case someone runs this from a single core machine.